### PR TITLE
Hide the ugly empty space above edit buttons if link is not multireddit.

### PR DESCRIPTION
**Pull request by pltarun**
Added :empty selector to the subreddit manager dropdown ul.

### CODE REVIEW

**Comment by pltarun** [comment]
http://i.imgur.com/f8RidFC.png
This is what I'm referring to.

**Comment by andytuba** [comment]
:+1: 

When this module gets a larger refactor, e.g. a template for the dropdown menu and/or a redesign, then populating the dropdown should skip the `<ul>` if there aren't any subreddits. This PR is a good patch for the current system and can easily be applied to RES v4.6.

File: lib/css/modules/_subredditManager.scss [hunk 1]
```diff
@@ -218,6 +218,10 @@
 				background-color: rgba(255, 255, 255, .6);
 			}
 		}
+
+		&:empty {
+			display: none;
```
Comment (andytuba) [inline lib/css/modules/_subredditManager.scss]: For bonus aesthetics, nix the extra line:

```css
& + .RESShortcutsEditButtons {
    border-top-width: 0; 
}
```

**Comment by pltarun** [comment]
I've kept the border and moved it to the ul element instead of .RESShortcutsEditButtons so that it is hidden when empty.

File: lib/css/modules/_subredditManager.scss [hunk 2]
```diff
@@ -205,6 +204,7 @@
 		overflow-y: auto;
 		max-width: 300px;
 		max-height: 500px;
+		border-bottom: 1px rgba(0, 0, 0, .5) solid;
```
Comment (andytuba) [inline lib/css/modules/_subredditManager.scss]: Sorry to throw you another curveball, but will this end up with a double border (ul & dropdown container) if the [dropdownEditButton](https://www.reddit.com/#!settings/subredditManager/dropdownEditButton) option is turned off? Could mitigate it by adding `bodyClass: true` to the `dropdownEditButton` option config and wrapping this `border-bottom` property in `.res-subredditManager-dropdownEditButton & { ... }`

**Comment by pltarun** [comment]
I think this is a better solution, as it ensures that the fix is pure CSS.

**Comment by andytuba** [comment]
Looks like you've covered all the edge cases I can think of now. Thanks for collaborating on this little fixup!

_waits patiently for travis green.._

**Pull request merged**